pkg/universe.dagger.io/x/david@rawkode.dev/kubernetes/kubectl.cue

```diff
@@ -0,0 +1,43 @@
+package kubectl
+
+import (
+	"dagger.io/dagger"
+	"universe.dagger.io/docker"
+)
+
+#Apply: {
+	manifests:        dagger.#FS
+	kubeconfigSecret: dagger.#Secret
+
+	// Kubernetes version
+	version: string | *"latest"
```

Comment:
`version` should always have a stable version as default so users will not have unstable behaviour when `kubectl` will be updated

Reply:
Good improvement for later!

Merging these learnings in the official `kubernetes` package, when we get to it, will be worth it.